core/src/options/input_variant.rs

```diff
@@ -95,6 +99,22 @@ impl ParseAttribute for InputVariant {
             }
 
             self.skip = FromMeta::from_meta(mi)?;
+        } else if path.is_ident("word") {
+            if self.word.is_some() {
+                return Err(Error::duplicate_field_path(path).with_span(mi));
+            }
+
+            if !self.data.is_unit() {
+                let note = "`#[darling(word)]` can only be applied to a unit variant";
```

Comment:
Nit: In the other error message, we didn't include the `#[darling(...)]` shell; should we also exclude it here? I forget what serde and the standard library do, so if you've got an example to help break the tie one way or the other that would be handy.